Codegen/test/resources/output/TypeCobol/CopyReplace4Colon/CopyReplace4Colon6.rdz.cbl

```diff
@@ -0,0 +1,11 @@
+﻿       IDENTIFICATION DIVISION.
+       PROGRAM-ID. CPYRPL4C.
+      
+       ENVIRONMENT DIVISION.
+       DATA DIVISION.
+       WORKING-STORAGE SECTION.
+       01 xxxENT2. COPY YxxxENT REPLACING ==::== BY ====.
+       01 xxxENT3. COPY YxxxENT REPLACING ==::== BY ==S==.
+       PROCEDURE DIVISION.
+           MOVE 'A' TO xxxENTS-FCT01-Var1.
+       END PROGRAM CPYRPL4C.
```

Comment:
Not much, but for this test we could simply use the following working-storage
```cob 
01 xxxENT. COPY YxxxENT REPLACING ==::== BY ==S==.
```

Reply:
The goal here is to test multiple COPY inclusion.